package.json

```diff
@@ -10,7 +10,7 @@
     "node": ">= 14"
   },
   "scripts": {
-    "postinstall": "husky install && yarn integrate",
+    "postinstall": "yarn integrate && if test -d .git; then husky install; fi",
```

Comment:
This was needed because I do not copy _everything_ to Docker container, and otherwise it would complain "it's not a Git project".